src/util/sample.h

```diff
@@ -27,7 +27,7 @@ class SampleUtil {
 
     // Allocated a buffer of CSAMPLE's with length size. Ensures that the buffer
     // is 16-byte aligned for SSE enhancement.
-    static CSAMPLE* alloc(SINT size);
+    [[nodiscard]] static CSAMPLE* alloc(SINT size);
```

Comment:
This is a welcome place for [[nodiscard]] btw.

Reply:
Indeed.
Having `[[nodiscard]]` has an habit will likely save you more than it'll cost you.

Reply:
I've removed them.